src/app/capture/page.tsx

```diff
@@ -6,9 +6,39 @@ import styles from './page.module.css';
 import DeerCamera from '@/components/deer-camera/DeerCamera';
 import Loader from '@/components/loader/Loader';
 import BottomNav from '@/components/bottom-nav/BottomNav';
+import { cooldownTime } from '@/utils/constants';
+import { useMutation } from '@tanstack/react-query';
+import { createDeerSighting } from '@/hooks/apis/useDeerSighting';
+import { useLocationContext } from '@/context/LocationContext';
 
 export default function Capture() {
   const [loading, setLoading] = useState(true);
+  const { userLocation } = useLocationContext();
+
+  const useCreateDeerSighting = useMutation({
+    mutationFn: createDeerSighting,
+    onSuccess: (res) => {
+      console.log('Deer Saved to System');
+      console.log(res);
+    },
+    onError: (err) => {
+      console.log(err);
+    },
+  });
```

Comment:
Do we wanna push this console.log type of error handling to prod? and do you think we can handle this error differently in the future and maybe creating an exception log? if you think so we might wanna some stories for that

Reply:
`console.log('Deer Saved to System')` will have to be removed when we go to prod. as for the rest to handle an error in typescript its most ideal to use `console.error`, instead of throwing an exception. I can go ahead and actually adjust the code in the error case in that regard and add `console.error` instead